### DISCUSSION TITLE
TraceCompleteness

### DISCUSSION THREAD

**Discussion by SwanHtetAung** [open-telemetry/opentelemetry-go#5449] [Q&A] [answered]
Hi everyone, 
Is there any existing work here to measure trace completeness? I have been looking through the OtelGo code and see that Otel Span Count has a [childSpanCount](https://github.com/open-telemetry/opentelemetry-go/blob/1bcb778c28deecded2b3425d8991d8ec73987645/sdk/trace/span.go#L711) which measures how many children the span should have. However, this only works for in-process spans and cannot count for the child span created due to RPC. 

More specifically, I wonder if there is any Otel standard guideline that measures trace quality and completeness.

**Comment by dmathieu** [comment, marked as answer]
If any work were to happen here, it would first have to be an [otep](https://github.com/open-telemetry/oteps/), and then a [specification](https://github.com/open-telemetry/opentelemetry-specification/).

However, I think this is something that would have to happen on the vendor side, not within your application itself.
